app/views/play/cinematic/CinematicLankBoss.js

```diff
@@ -68,8 +103,10 @@ export default class CinematicLankBoss {
       y: pos.y,
       duration: ms,
       autoplay: false,
-      // Update required to ensure Lank sprite position is moved.
-      update: () => { this[side].thang.stateChanged = true }
+      easing: 'easeInOutQuart',
```

Comment:
What is this for?

Reply:
The default animation ease function has bounce which makes the characters bounce around a little bit too much. This is a much more pleasing easing function.